.gitignore

```diff
@@ -81,3 +81,4 @@ lint/tmp/
 
 # Android Profiling
 *.hprof
+samples/kotlin_compose/src/google-services.json
```

Comment:
due to nested `gitignore` this file was being tracked no matter what i did. So ignroing it in root gitignore